From: David Cohen <david.a.cohen@intel.com>
Subject: [PATCH] usb: dwc3: REVERTME: temporarily disable pm runtime autosuspend

PM runtime autosuspend seems to be causing rare stability issues. While we
investigate why, it will stay disabled.

Signed-off-by: David Cohen <david.a.cohen@intel.com>

---
 drivers/usb/dwc3/core.c |    8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

--- a/drivers/usb/dwc3/core.c
+++ b/drivers/usb/dwc3/core.c
@@ -806,7 +806,13 @@ static int dwc3_probe(struct platform_de
 
 	if (dwc->runtime_suspend) {
 		pm_runtime_set_autosuspend_delay(dev, 10000);
-		pm_runtime_use_autosuspend(dev);
+		
+		/*
+		 * Autosuspend seems to be the root cause of few rare issues
+		 * on this driver. We're disabling it while investigate why.
+		 */
+		/* pm_runtime_use_autosuspend(dev); */
+		
 		pm_runtime_mark_last_busy(dev);
 		pm_runtime_put_autosuspend(dev);
 
